src/dialect/postgresql.rs

```diff
@@ -209,6 +214,11 @@ impl Dialect for PostgreSqlDialect {
     fn supports_comment_on(&self) -> bool {
         true
     }
+
+    /// See <https://www.postgresql.org/docs/current/sql-load.html>
+    fn supports_load_extension(&self) -> bool {
+        true
+    }
```

Comment:
Oh, I'm not sure I followed this change, I can see this flag and tests for it but there doesn't seem to be any changes to the parser itself?

Reply:
As I understand it, this change simply adds support for the `LOAD  extension` statement to the `PostgreSQL` dialect, while the  `DuckDB` and `Generic` dialects both already support it. Therefore, no changes are needed to the parser. In my local environment, this modification is sufficient. If my understanding is incorrect, please feel free to correct me. ：）

Reply:
Ah yes ofc my bad! I missed that the feature already existed and was just being enabled for postgres :+1: